### ISSUE TITLE
resnet_model maybe wrong in resnet_v1

### ISSUE THREAD

**Issue by irmowan** [tensorflow/models#4198]
When I use building_block_v1 or bottleneck_block_v1, the last several layers are:

block: Conv, Batch Norm, add, ReLU, (https://github.com/tensorflow/models/blob/master/official/resnet/resnet_model.py#L143
)
then, Batch Norm, ReLU (https://github.com/tensorflow/models/blob/master/official/resnet/resnet_model.py#L521)

This seems wrong. The last BN and ReLU appears only in Resnet_v2.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by qlzh727** [comment]
I think you are correct. V2 need the final BN and Relu activation at the end since each block was doing pre activation. Will send a fix very soon. 

Also adding @robieta to this thread since he originally implemented this model.

**Comment by robieta** [comment]
Great catch!

@karmel @tfboyd 
You know that elusive difference between official/resnet and tf_cnn_benchmarks/resnet?

https://github.com/tensorflow/benchmarks/blob/master/scripts/tf_cnn_benchmarks/models/resnet_model.py#L202

https://github.com/tensorflow/benchmarks/blob/master/scripts/tf_cnn_benchmarks/models/resnet_model.py#L222

**Comment by qlzh727** [comment]
Pending fix in #4230

**Comment by qlzh727** [comment]
Should be fixed by #4230

**Comment by irmowan** [comment]
Checked https://github.com/tensorflow/models/pull/4230, still wrong.
Resnet_v1 first conv layer without BN and relu.
Refer to https://github.com/KaimingHe/deep-residual-networks/blob/master/prototxt/ResNet-50-deploy.prototxt

**Comment by qlzh727** [comment]
Thanks for reporting the issue. We are looking at the reference and see why it is missing in the first place. We think V1 will need that BN and relu, and investigating the V2 structure.
